sdk/assemblyscript/examples/auth/assembly/index.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * This example is part of the Modus project, licensed under the Apache License 2.0.
+ * You may modify and use this example in accordance with the license.
+ * See the LICENSE file that accompanied this code for further details.
+ */
+
+import { auth } from "@hypermode/modus-sdk-as";
+
+
+@json
+export class Claims {
+  public exp!: i64;
+  public iat!: i64;
+  public iss!: string;
+  public jti!: string;
+  public nbf!: i64;
+  public sub!: string;
+
+
+  @alias("user-id")
+  public userId!: string;
+}
```

Comment:
It's unclear that this is an _example_ of a claims object you might want to retrieve.  One might mistakenly think they always needed exactly this class structure.  Some comments would help.  (Same for the Go example)